test/models/pusher_test.rb

```diff
@@ -499,7 +499,7 @@ def two_cert_chain(signing_key:, root_not_before: Time.current, cert_not_before:
     should "be false if rubygem is new and api key has unexpected owner type" do
       @cutter.stubs(:rubygem).returns Rubygem.new
 
-      owner = stub("owner")
+      owner = stub("owner", to_gid: nil)
```

Comment:
Nit: since the gid isn't intentionally nil, you just want it to not raise, set it to maybe `:to_gid`?